### ISSUE TITLE
Incorrect MusicXML element name

### ISSUE THREAD

**Issue by twcook** [cuthbertLab/music21#174]
When writing to MusicXML with a Baritone Saxophone (or any other instrument requiring an octave change), the transpose complexType  
http://usermanuals.musicxml.com/MusicXML/Content/CT-MusicXML-transpose.htm does not define an element <octave-shift> it should be <octave-change>

Sample Music XML snippet from music21: 
````
<measure number="0">
      <attributes>
        <divisions>10080</divisions>
        <transpose>
          <diatonic>-26</diatonic>
          <chromatic>3</chromatic>
          <octave-shift>-2</octave-shift>
        </transpose>
        <time>
          <beats>4</beats>
          <beat-type>4</beat-type>
        </time>
      </attributes>

````

octave-shift is a separate complexType defined here:
http://usermanuals.musicxml.com/MusicXML/Content/CT-MusicXML-octave-shift.htm

**Comment by twcook** [comment]
Apologies I could get the code markdown to work correctly.

**Comment by mscuthbert** [comment]
Are you on the latest github pull of music21?  I did some changes to this a few months ago which are slated for the v. 3 release and I just want to make sure that this wasn't already fixed.  Thanks!

**Comment by twcook** [comment]
Yep, a fresh pull using commit 16e9f28c608a1ce7ba7cb60ad2048a708627da73 has this error still. 
Thanks.

**Comment by mscuthbert** [comment]
Ah! Now I see what you meant by the error.  Yes! Fixing now.

**Comment by twcook** [comment]
Oops! I just reread my issue post and realize I deleted part of the note when I pasted it.  :-(  
Glad you figured out my cryptic writing.

**Comment by mscuthbert** [comment]
Fixed in 8ed7e59  Thanks!
